### PR TITLE
First draft of release notes for DC/OS 1.9.7.

### PR DESCRIPTION
**Pull request by pavisandhu**
## Description
https://jira.mesosphere.com/browse/RELEASE-434

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.9/release-notes/1.9.7/index.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+# Issues Fixed in DC/OS 1.9.6
+
+- COPS-980 - Marathon's default task unreachable behavior reinstated.
+- DCOS-17947 - [`cluster_docker_credentials`](/1.9/installing/oss/custom/configuration/examples/#dcos-cluster-with-three-masters-an-exhibitorzookeeper-managed-internally-custom-docker-credentials-two-private-agents-and-google-dns) example corrected.
+- DCOS-19217 - Metronome adheres to crontab standard.
+- DCOS-19453 - TLS compression disabled because TLS is vulnerable to the CRIME attack. [enterprise type="inline" size="small" /]
+- DCOS-19508 - Non-superusers can now edit an app that uses secrets. [enterprise type="inline" size="small" /]
+- DOCS-2130 - [Security hardening guide](/1.9/security/ent/hardening/) improvements. [enterprise type="inline" size="small" /]
+- DOCS-2153	- [Metrics documentation](/1.9/metrics/) improvements.
```
Comment (ichernetsky) [inline pages/1.9/release-notes/1.9.7/index.md]: nit: too many spaces after the ticket number.

File: pages/1.9/release-notes/1.9.7/index.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+---
+
+These are the release notes for DC/OS 1.9.7.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.7/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+# Issues Fixed in DC/OS 1.9.7
+
+- CORE-1375 - Docker executor no longer hangs due to lost messages.
+- DCOS_OSS-1943 - Modified the LIBPROCESS_WORKER_THREADS environment variable to 16 to increase the minimum number of worker threads.
```
Comment (elezar) [inline pages/1.9/release-notes/1.9.7/index.md]: Should this be: `LIBPROCESS_WORKER_THREADS` (with back-ticks)?

File: pages/1.9/release-notes/1.9.7/index.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+# Notable Changes in DC/OS 1.9.7
+
+- Mesos 1.2.3 is integrated with DC/OS 1.9.7.
```
Comment (elezar) [inline pages/1.9/release-notes/1.9.7/index.md]: Should this be "has been"? In previous releases (see below, a link to the relevant changelog was provided) and the wording was different.

File: pages/1.9/release-notes/1.9.7/index.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+excerpt:
+---
+
+These are the release notes for DC/OS 1.9.7.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.7/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+# Issues Fixed in DC/OS 1.9.7
+
+- CORE-1375 - Docker executor no longer hangs due to lost messages.
```
Comment (elezar) [inline pages/1.9/release-notes/1.9.7/index.md]: nit: Although it seems that the standard is to end every item on a period, common practice is to only do this if the item is a complete sentence.

**Comment by marcomonaco** [comment]
Missing new Metronome release 0.3.2

File: pages/1.9/release-notes/1.9.7/index.md (hunk 1 above)
Comment (pavisandhu) [reply pages/1.9/release-notes/1.9.7/index.md]: Formatting is consistent with previous versions of release notes.

File: pages/1.9/release-notes/1.9.7/index.md (hunk 4 above)
Comment (pavisandhu) [reply pages/1.9/release-notes/1.9.7/index.md]: Formatting is consistent with previous versions of release notes.

**Pull request merged**